testing/fleetservertest/fleetserver_test.go

```diff
@@ -40,28 +146,523 @@ func ExampleNewServer_status() {
 
 func ExampleNewServer_checkin() {
 	agentID := "agentID"
+	tmpl := TmplPolicy{ActionID: "anActionID"}
+
+	actions, err := NewActionPolicyChangeWithFakeComponent(tmpl)
```

Comment:
I think this just returns a single action?

```suggestion
	action, err := NewActionPolicyChangeWithFakeComponent(tmpl)
```